### PR TITLE
Profile orders show and index

### PR DESCRIPTION
**Pull request by janegreene**
Removed two migrations causing conflicts
Created views for orders show and orders index where all details can be viewed.
Created method of quantity to order model

### CODE REVIEW

File: app/views/orders/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+  <p>Created: <%= order.created_at%></p>
+</section>
+<section id="dateupdated">
+  <p> Last Updated: <%= order.updated_at%></p>
+</section>
+
+<section id="grandtotal">
+  <p>Grand Total: <%=number_to_currency(order.grandtotal)%></p>
+</section>
+</section>
```
Comment (jpc20) [inline app/views/orders/index.html.erb:31]: Nice, if we had more time we could have made this view a partial.

File: spec/features/orders/show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      expect(page).to have_content(seat.description)
+      expect(page).to have_css("img[src*='#{seat.image}']")
+      expect(page).to have_content(seat.price)
+      expect(page).to have_content(item_order2.subtotal)
+    end
+    within("#datecreated") do
+      expect(page).to have_content("Created: #{order1.created_at}")
+    end
+    within("#dateupdated") do
+      expect(page).to have_content("Last Updated: #{order1.updated_at}")
```
Comment (Lithnotep) [inline spec/features/orders/show_spec.rb:49]: solid tests here

File: spec/features/orders/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    within("#item-#{item_order2.item_id}") do
+      expect(page).to have_content("Seat")
+      expect(page).not_to have_content("Gatorskins")
+      expect(page).to have_content(seat.description)
+      expect(page).to have_css("img[src*='#{seat.image}']")
+      expect(page).to have_content(seat.price)
+      expect(page).to have_content(item_order2.subtotal)
+    end
+    within("#datecreated") do
+      expect(page).to have_content("Created: #{order1.created_at}")
```
Comment (jpc20) [inline spec/features/orders/show_spec.rb:46]: Lets find out how to stub the date in RSpec

File: spec/features/users/user_profile_show_spec.rb [hunk 4]
```diff
@@ -170,5 +170,53 @@
     expect(page).to have_content("Your password has been changed!")
 
   end
-
+  it "displays orders link" do
+    @user = create(:user)
+    bike_shop = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
```
Comment (Ashkanthegreat) [inline spec/features/users/user_profile_show_spec.rb:175]: Move these in a "before each"?

**Pull request merged**